src/EditorFeatures/CSharpTest/Formatting/CodeCleanupTests.cs

```diff
@@ -756,7 +756,7 @@ public async Task DoNotApplyFixerIfChangesAreMadeOutsideDocument()
             var enabledDiagnostics = codeCleanupService.GetAllDiagnostics();
 
             var newDoc = await codeCleanupService.CleanupAsync(
-                document, enabledDiagnostics, new ProgressTracker(), options, CancellationToken.None);
+                document, enabledDiagnostics, CodeAnalysisProgress.None, options, CancellationToken.None);
```

Comment:
CodeAnalysisProgress.None is used for code that has no way to display UI when calling into apis that can report progress. It should be used sparingly (using in tests is fine). In product code, it represents cases where teh user may be waiting a long time, with no knowledge of what's going on.